evm-contracts/src/v0.7/dev/Operator.sol

```diff
@@ -302,64 +299,61 @@ contract Operator is
     // Replace list
     s_authorizedSenderList = senders;
     emit AuthorizedSendersChanged(senders);
-
-    // Set authorized senders on the forwarder
-    address[] memory forwarderSenders = new address[](senders.length+1);
-    for (uint256 i = 0; i < senders.length; i++) {
-      forwarderSenders[i] = senders[i];
-    }
-    forwarderSenders[senders.length] = msg.sender;
-    s_forwarder.setAuthorizedSenders(forwarderSenders);
   }
 
   /**
-   * @notice If the s_forwarder is owned by a different address, deploy and set a new one
+   * @notice Sets the fulfillment permission for
+   * @param senders The addresses that are allowed to send updates
+   * @param targets The addresses to set permissions on
    */
-  function deployForwarder()
+  function setAuthorizedSendersOn(
+    address[] calldata senders,
```

Comment:
Bit of a nitpick, but feels like `targets` should be ordered before `senders`. "For each target, set these senders".